interfaces/builtin/opengl.go

```diff
@@ -51,7 +51,7 @@ const openglConnectedPlugAppArmor = `
 
   # FIXME: this is an information leak and snapd should instead query udev for
   # the specific accesses associated with the above devices.
-  /sys/bus/pci/devices/** r,
+  /sys/bus/pci/devices/ r,
```

Comment:
Has this change been tested?  Why was the prior case necessary and why is it not needed anymore now?

This also seems unrelated to device tagging.

Reply:
I reverted this change now - please open a new PR with this particular change. We are keen to land this one.

Reply:
Sure, no problem.:)